Generators/src/main/java/net/minestom/code_generation/AllGenerators.java

```diff
@@ -0,0 +1,112 @@
+package net.minestom.code_generation;
```

Comment:
The main reasoning to give it code_generation is that using "generated" or "generation" could be misinterpreted with terrain generation for example. I can change it but since this package is in it's own subproject I don't think this is a necessary change  but it's a simple refractor in IntelliJ if you insist we can change this :).